Revit_Core_Engine/Query/BuiltInCategories.cs

```diff
@@ -48,6 +55,11 @@ public static HashSet<BuiltInCategory> BuiltInCategories(this oM.Adapters.Revit.
 
         /***************************************************/
 
+        [Description("Queries a BHoM generic object for its Revit BuiltInCategory.")]
```

Comment:
Generic is a risky word in C# 😉 I would get rid of it for clarity, the message is still informative enough I think.

```suggestion
        [Description("Queries a BHoM object for its Revit BuiltInCategory.")]
```